front/package.json

```diff
@@ -48,7 +48,7 @@
   "dependencies": {
     "@apollo/client": "^3.5.6",
     "@babel/runtime": "7.x",
-    "@citizenlab/cl2-component-library": "^0.10.17",
+    "@citizenlab/cl2-component-library": "^0.10.19",
```

Comment:
Includes a component library improvement where now the link in the tooltip with black background is white, instead of blue.